firefox/guide/firefox/firefox_preferences-background_data/rule.yml

```diff
@@ -0,0 +1,29 @@
+documentation_complete: true
+
+title: 'Background submission of information to Mozilla must be disabled.'
```

Comment:
Please make the title:
`Disable Background Information Submission`

Reply:
Edited the title accordingly